k8s/solution/answers.md

```diff
@@ -0,0 +1,26 @@
+# Lesson 1
+
+> Do you see any difference when using `kubectl` to delete the pod? 
+
+* Using the "Shutdown" button the application terminates itself, putting the pod into `completed` state.
+  After this the app is restarted within the *same* pod.
+* Killing a pod with `kubectl delete pod` kills the application and removes the pod from the list while
+  spawning a new pod (= application instance).
+
+
+
+
+
+hmm - das war's irgendwie mit den Fragen, vielleicht ist die answers.md doch ein bisschen überflüssig
+
+
+# Lesson 4
+
+> ... bad responses on the `/status` endpoint. Watch again what happens with the pods.
+
+Nothing happens?!
```

Comment:
Did you wait long enough? It only checks every 5 seconds or so, and I think it might have a tolerance of one of them failing as well (not sure)
Maybe the new pod spawned too fast for you to see it. I could see i in the Restart count.